packages/fluent-theme/src/components/sendbox/index.module.css

```
@@ -25,6 +36,22 @@
       - we are using colorCompoundBrandForeground1Hover */
     box-shadow: inset 0 -6px 0 -3px var(--webchat-colorCompoundBrandForeground1Hover);
   }
+
+  > .sendbox__text-area--in-grid {
```

Maybe we don't need "immediate child" because we are BEM?

```suggestion
  .sendbox__text-area--in-grid {
```

---

We want to indicate that this selector will work only on direct children

---

It's good. I thought if it will be too fragile. I guess not because this is a grid and supposedly to be direct children.